test_app/Dockerfile

```diff
@@ -7,15 +7,18 @@ RUN apt-get update -y && \
     apt-add-repository -y ppa:brightbox/ruby-ng && \
     apt-get update -y
 
-RUN apt-get install -y ruby2.4 ruby2.4-dev python build-essential
+RUN apt-get install -y ruby2.4 ruby2.4-dev python && apt-get update && apt-get install -y gcc build-essential
 
 RUN gem install bundler --no-rdoc --no-ri
 
 RUN mkdir -p /opt/app
 WORKDIR /opt/app
 
 COPY Gemfile /opt/app/
-COPY Gemfile.lock /opt/app/
 RUN bundle install
 
 COPY . /opt/app
+
+RUN mkdir -p /usr/local/lib/summon
+COPY summon-conjur /usr/local/lib/summon
```

Comment:
I'd strong prefer to get this moved into our Ansible Role.  It feels like cheating to copy it into the container. :)

Reply:
@jvanderhoof i've gone ahead and used the `summon` and `summon-conjur` install scripts in the playbook. i think that's a bit more fair. 

here's a thought! users might not trust running a script that calls out to the net to install summon and so might opt to build the binaries themselves and just have a step in the playbook to copy them into the remote hosts.